### PR TITLE
[doc] corrections for Yolo V1 and V2 coversion

### PR DESCRIPTION
**Pull request by pavel-esir**
### Details:
- cherry-pick of #6283
 - added some correction to successfully export from DarfkFlow to TensorFlow frozen model

### Tickets:
 - 55341, 55346

### CODE REVIEW

File: docs/MO_DG/prepare_model/convert_model/tf_specific/Convert_YOLO_From_Tensorflow.md [hunk 1]
```diff
@@ -144,9 +144,22 @@ cd darkflow
...
+python3 flow --model yolov1.cfg --load yolov1.weights --savepb
+```
+
+For Yolo V2 with VOC dataset additional changes in the original exporting script are required. 
+In the file`darkflow/utils/loader.py` change line 121 from `self.offset = 16` to `self.offset = 20` 
+And then run:
+```sh
+python3 flow --model yolov2-voc.cfg --load yolov2-voc.weights --labels voc-labels.txt --savepb
+```
+voc-labels can be found by the following link https://raw.githubusercontent.com/szaza/android-yolo-v2/master/assets/tiny-yolo-voc-labels.txt
```
Comment (yekruglov) [inline docs/MO_DG/prepare_model/convert_model/tf_specific/Convert_YOLO_From_Tensorflow.md:161]: Should not we mark this as hyperlink? Something like

`voc-labels can be found by the following [link](https://raw.githubusercontent.com/szaza/android-yolo-v2/master/assets/tiny-yolo-voc-labels.txt)`

Comment (pavel-esir) [reply docs/MO_DG/prepare_model/convert_model/tf_specific/Convert_YOLO_From_Tensorflow.md:161]: The link is not 'official' so I wanted user to the full path. 
If i don't write [](https://) Markdown still shows it as a link but I can explicitly specify 
```[https://raw.githubusercontent.com/szaza/android-yolo-v2/master/assets/tiny-yolo-voc-labels.txt](https://raw.githubusercontent.com/szaza/android-yolo-v2/master/assets/tiny-yolo-voc-labels.txt)```


if we replace to see a short link, e.g. ```[short_link_name](https://raw.githubusercontent.com/szaza/android-yolo-v2/master/assets/tiny-yolo-voc-labels.txt)``` I suspect user may scare that we are trying to 'fish` him. Do you insist me to write a short link?

**Review by yekruglov — APPROVED** [review]

**Review by yekruglov — APPROVED** [review]

**Review by achetver — APPROVED** [review]
Looks good to me.

**Review by evolosen — APPROVED** [review]

File: docs/MO_DG/prepare_model/convert_model/tf_specific/Convert_YOLO_From_Tensorflow.md [hunk 2]
```diff
@@ -143,11 +143,23 @@ cd darkflow
...
 
 To convert YOLOv1 or YOLOv2 model to TensorFlow, go to the root directory of the cloned DarkFlow repository and run the following command:<br>
+For Yolo V1:
 ```sh
-python3 ./flow --model <path_to_model>/<model_name>.cfg --load <path_to_model>/<model_name>.weights --savepb
+python3 flow --model yolov1.cfg --load yolov1.weights --savepb
 ```
 
-If the model was successfully converted, you can find the `<model_name>.meta` and `<model_name>.pb` files
+For Yolo V2 with VOC dataset `--labels` argument should be specified and additional changes in the original exporting script are required.
```
Comment (lazarevevgeny) [inline docs/MO_DG/prepare_model/convert_model/tf_specific/Convert_YOLO_From_Tensorflow.md:151]: Add commit hash.

**Review by lazarevevgeny — APPROVED** [review]
Only one comment. The rest looks good.

File: docs/MO_DG/prepare_model/convert_model/tf_specific/Convert_YOLO_From_Tensorflow.md (hunk 2 above)
Comment (pavel-esir) [reply docs/MO_DG/prepare_model/convert_model/tf_specific/Convert_YOLO_From_Tensorflow.md:151]: answered in the PR for master

Comment (pavel-esir) [reply docs/MO_DG/prepare_model/convert_model/tf_specific/Convert_YOLO_From_Tensorflow.md:151]: done

**Comment by pavel-esir** [comment]
please **do not merge** until July 5.

File: docs/MO_DG/prepare_model/convert_model/tf_specific/Convert_YOLO_From_Tensorflow.md [hunk 3]
```diff
@@ -143,11 +143,24 @@ cd darkflow
 #### <a name="yolov1-v2-to-tf"></a>Convert DarkNet\* YOLOv1 or YOLOv2 Model to TensorFlow\*
 
 To convert YOLOv1 or YOLOv2 model to TensorFlow, go to the root directory of the cloned DarkFlow repository and run the following command:<br>
+For Yolo V1:
 ```sh
-python3 ./flow --model <path_to_model>/<model_name>.cfg --load <path_to_model>/<model_name>.weights --savepb
+python3 flow --model yolov1.cfg --load yolov1.weights --savepb
```
Comment (ncybulsk) [inline docs/MO_DG/prepare_model/convert_model/tf_specific/Convert_YOLO_From_Tensorflow.md:148]: I would mention that user needs to place model files in the same directory for this command to work

Comment (pavel-esir) [reply docs/MO_DG/prepare_model/convert_model/tf_specific/Convert_YOLO_From_Tensorflow.md:148]: done

**Comment by pavel-esir** [comment]
For some reason request re-review button is not working on this PR 
image

 @ncybulsk applied your comment, please review.

**Review by andrew-zaytsev — APPROVED** [review]

**Pull request merged**